<ui version="4.0" >
 <class>NVideoPlaybackSettings</class>
 <widget class="QWidget" name="NVideoPlaybackSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="NGradientLabel" name="titleLabel" >
     <property name="font" >
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip" >
      <string>Neuros Gradient QLabel</string>
     </property>
     <property name="whatsThis" >
      <string>Inherited from QLabel, plus the ability to display with gradient color.</string>
     </property>
     <property name="text" >
      <string>Play Mode Settings</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>49</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <spacer name="horizontalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>76</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout_7" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="repeatModeLabel" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>125</red>
               <green>125</green>
               <blue>125</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>125</red>
               <green>125</green>
               <blue>125</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>118</red>
               <green>116</green>
               <blue>113</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Repeat Mode</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="playModeLabel" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>125</red>
               <green>125</green>
               <blue>125</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>125</red>
               <green>125</green>
               <blue>125</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>118</red>
               <green>116</green>
               <blue>113</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Play Mode</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resumeLabel" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>125</red>
               <green>125</green>
               <blue>125</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>125</red>
               <green>125</green>
               <blue>125</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>118</red>
               <green>116</green>
               <blue>113</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Resume From Last Position</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="palybackProportionLabel" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>125</red>
               <green>125</green>
               <blue>125</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>125</red>
               <green>125</green>
               <blue>125</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>118</red>
               <green>118</green>
               <blue>117</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Playback Proportions</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="NHorizontalComboBox" name="repeatModeComboBox" >
           <property name="font" >
            <font>
             <pointsize>24</pointsize>
            </font>
           </property>
           <item>
            <property name="text" >
             <string>Normal</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Repeat</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Shuffle</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <item>
          <widget class="NHorizontalComboBox" name="playModeComboBox" >
           <property name="font" >
            <font>
             <pointsize>24</pointsize>
            </font>
           </property>
           <item>
            <property name="text" >
             <string>Single</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Folder</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>198</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" >
         <item>
          <widget class="NHorizontalComboBox" name="resumeComboBox" >
           <property name="font" >
            <font>
             <pointsize>24</pointsize>
            </font>
           </property>
           <item>
            <property name="text" >
             <string>Disabled</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Enabled</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" >
         <item>
          <widget class="NHorizontalComboBox" name="proportionComboBox" >
           <property name="font" >
            <font>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="frame" >
            <bool>true</bool>
           </property>
           <item>
            <property name="text" >
             <string>4:3 (Normal)</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>16:9 (Widescreen)</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="2" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>39</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>63</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>338</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="NPushButton" name="saveButton" >
         <property name="font" >
          <font>
           <pointsize>22</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Save</string>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NPushButton" name="exitButton" >
         <property name="font" >
          <font>
           <pointsize>22</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Exit</string>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <property name="spacing" >
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="NDateTimeLabel" name="datetimeLabel" >
       <property name="font" >
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="toolTip" >
        <string>Neuros Date and Time QLabel</string>
       </property>
       <property name="whatsThis" >
        <string>Inherited from QLabel, custom it for displaying the date and time.</string>
       </property>
       <property name="role" stdset="0" >
        <enum>NDateTimeLabel::TimeOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ximIconLabel" >
       <property name="minimumSize" >
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../resources/lib-gui.qrc" >:/lib-gui/xim-24x24.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NGradientLabel</class>
   <extends>QLabel</extends>
   <header>ngradientlabel.h</header>
  </customwidget>
  <customwidget>
   <class>NDateTimeLabel</class>
   <extends>QLabel</extends>
   <header>ndatetimelabel.h</header>
  </customwidget>
  <customwidget>
   <class>NHorizontalComboBox</class>
   <extends>QComboBox</extends>
   <header>nhorizontalcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>NPushButton</class>
   <extends>QPushButton</extends>
   <header>npushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>playModeComboBox</tabstop>
  <tabstop>resumeComboBox</tabstop>
  <tabstop>proportionComboBox</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/lib-gui.qrc" />
 </resources>
 <connections/>
</ui>
